charts/kyverno/crds/crds.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     shortNames:
     - cpol
     singular: clusterpolicy
-  scope: Cluster
+  scope: Namespaced
```

Comment:
We should support both cluster-wide policy and namespace policy. 
When adding support for namespaced policy, also need to cover:
- how the webhook handle namespaced policy?
- how generate policy works?
- do we want to update background logic (policy violation)?